app/adminequipmentapp/webapp/test/integration/opaTests.qunit.js

```
@@ -0,0 +1,26 @@
+sap.ui.require(
+    [
+        'sap/fe/test/JourneyRunner',
+        'com/equip/demo/adminequipmentapp/test/integration/FirstJourney',
+		'com/equip/demo/adminequipmentapp/test/integration/pages/EquipmentsList',
+		'com/equip/demo/adminequipmentapp/test/integration/pages/EquipmentsObjectPage'
+    ],
+    function(JourneyRunner, opaJourney, EquipmentsList, EquipmentsObjectPage) {
+        'use strict';
+        var JourneyRunner = new JourneyRunner({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'JourneyRunner'. Consider to delete it or rename it.

'JourneyRunner' is defined here:


(lint/suspicious/noRedeclare)